### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by mohitz007** [TeamVanced/VancedManager#483] [Ideas]
Can you add a loop button. Just like in desktop web version.
Thank in advance.

**Comment by KevinX8** [comment]
Wrong repo, and it's already a feature check vanced settings

**Comment by mohitz007** [comment]
Yeah. Found thanks

**Comment by drnico2016** [comment]
Every update are such a pain 😂😂😂
But the app when it work is good

**Reply by ImtiazTonmoy290** [reply]
Get the apk of YouTube. It won't auto update.
